bin/start_fe.sh

```diff
@@ -176,6 +176,12 @@ if [ ! -d $LOG_DIR ]; then
     mkdir -p $LOG_DIR
 fi
 
+meta_dir="$STARROCKS_HOME/meta"
+read_var_from_conf meta_dir $STARROCKS_HOME/conf/fe.conf
```

Comment:
just check if `meta_dir` is exported by line 69, and if not, use the default value, and then do the `mkdir`